integration/simulation/transaction_injector.go

```diff
@@ -179,7 +180,12 @@ func (m *TransactionInjector) issueRandomTransfers() {
 		signedTx := wallet_mock.SignTx(tx, fromWallet.Key.PrivateKey)
 		encryptedTx := core.EncryptTx(signedTx)
 		m.stats.Transfer()
-		m.rndL2Node().P2p.BroadcastTx(encryptedTx)
+
+		err := (*m.rndL2NodeClient()).Call(nil, obscuroclient.RPCSendTransactionEncrypted, encryptedTx)
+		if err != nil {
+			continue
```

Comment:
Shouldn't we at least log transactions that failed to inject or is there some reason we'll expect to see regular failures?

Reply:
I reckon we'll see a few after the clients/nodes have been stopped. This will be picked up anyway in the tests at the end.

Reply:
I'll add some logging locally, and see how noisy it is.

Reply:
It's not noisy at all actually - logs added :)

Reply:
Yeah fair, can always dial it back but at least this way if it breaks unexpectedly it'll be easier to debug than tests randomly failing